### PR TITLE
Moar export! Add pregnancy fields, clinic name, age range

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

@cfinucane you built a really remarkably extendable baseboard for the export function, color me impressed!, and also grateful! This adds fields I think we definitely want to export from patient, but please check my work.

It removes special circumstances and switches age to a range as well, and adds clinic info.

This pull request makes the following changes:
* Adds pregnancy fields
* Trims or adjusts some others per conversation in #812 
* Adds a model for getting at related objects

Still TK, handling calls and fulfillments.

It relates to the following issue #s: 
* Bumps #965 

@cfinucane @lomky for review

### CODE REVIEW

File: app/models/concerns/exportable.rb [hunk 1]
```diff
@@ -21,16 +21,56 @@ module Exportable
...
 
+  def export_clinic_name
+    clinic.try :name
+  end
+
+  def age_range
+    case age
+    when nil
+      nil
+    when 0..18
```
Comment (lomky) [inline app/models/concerns/exportable.rb]: `0..17`?

File: app/models/concerns/exportable.rb [hunk 2]
```diff
@@ -21,16 +21,56 @@ module Exportable
...
-    "Special Circumstances" => :special_circumstances 
-  }
+    # "Special Circumstances" => :special_circumstances,
+    "LMP at intake (weeks)" => :last_menstrual_period_weeks,
+    "Race or Ethnicity" => :race_ethnicity,
+    "Employment status" => :employment_status,
+    "Abortion cost" => :procedure_cost,
+    "Patient contribution" => :patient_contribution,
+    "NAF pledge" => :naf_pledge,
+    "Fund pledge" => :dcaf_soft_pledge,
```
Comment (lomky) [inline app/models/concerns/exportable.rb]: assuming this is after #1061, this will need a bump

File: app/models/concerns/exportable.rb [hunk 3]
```diff
@@ -21,16 +21,56 @@ module Exportable
...
+    "Clinic" => :export_clinic_name,
+    "Pledge sent" => :pledge_sent,
+    "Resolved without fund assistance" => :resolved_without_dcaf,
```
Comment (lomky) [inline app/models/concerns/exportable.rb]: assuming this is after #1061, this will need a bump

File: app/models/concerns/exportable.rb [hunk 4]
```diff
@@ -21,16 +21,56 @@ module Exportable
...
+      '18-24'
+    when 25..34
+      '25-34'
+    when 35..44
+      '35-44'
+    when 45..54
+      '45-54'
+    when 55..100
+      '55+'
+    end
```
Comment (DarthHater) [inline app/models/concerns/exportable.rb]: While improbable you could likely do a else case to handle a typo. Suggest 'Centarian' :)

File: app/models/concerns/exportable.rb (hunk 1 above)
Comment (DarthHater) [inline app/models/concerns/exportable.rb]: Damn! Lomky with the good eyes

File: app/models/concerns/exportable.rb (hunk 4 above)
Comment (colinxfleming) [inline app/models/concerns/exportable.rb]: The test is a good flag! So is the default case. I made it 'bad value' which seemed okay.

Comment (colinxfleming) [inline app/models/concerns/exportable.rb]: Resolved in ded1e93

File: app/models/concerns/exportable.rb (hunk 1 above)
Comment (colinxfleming) [inline app/models/concerns/exportable.rb]: oh man, great, great catch.

File: app/models/concerns/exportable.rb (hunk 3 above)
Comment (colinxfleming) [inline app/models/concerns/exportable.rb]: Bumped in 5866b74

**Comment by colinxfleming** [comment]
Removed `identifer` export in aea08ab5ca43ca5b85c0a8b43bbd1ad673fd6aa9 . Thanks all for the sharp and thoughtful feedback.

**Pull request merged**